awx/main/tasks.py

```diff
@@ -956,11 +956,11 @@ def build_passwords(self, job, **kwargs):
         and ansible-vault.
         '''
         passwords = super(RunJob, self).build_passwords(job, **kwargs)
-        for cred, fields in {
-            'credential': ('ssh_key_unlock', 'ssh_password', 'become_password'),
-            'vault_credential': ('vault_password',)
+        for kind, fields in {
+            'ssh': ('ssh_key_unlock', 'ssh_password', 'become_password'),
+            'vault': ('vault_password',)
```

Comment:
This is another place we'll extend for multi-vault support w/ prompted passwords indexed by `--vault-id`

Reply:
This also paves the way if we ever support multi-machine creds in the future.